mintlify-docs/router/configuration/template-expressions.mdx

```diff
@@ -1,105 +1,110 @@
+
+```yaml
 security:
   block_mutations:
     enabled: true
-    condition: "request.header.Get('x-block-mutation') == 'yes'"
-```
-
-In this example, the condition is evaluated each time a request is made. Template Expressions provide access to request details, including headers, URLs, and query parameters. However, not all fields are consistently available throughout the request lifecycle. For instance, authentication data becomes accessible only after successful authentication.
-
-During the build process, the router validates expressions. If an expression is invalid or does not return the expected type (such as a boolean or string), an error message is generated to ensure clarity and correctness.
-
-### Template Language
-
-The expressions are based on the [expr-lang](https://github.com/expr-lang/expr) template language. This language is characterized by its:
-
-* **Safety** – Expressions are evaluated securely.
-
-* **Speed** – The language is optimized for performance.
-
-* **Side-effect-free operation** – Expressions are read-only and do not alter state.
-
-To modify configurations or manipulate requests and responses, implementing a [custom module](/router/custom-modules) is recommended.
-
-#### Naming Conventions
-
-* **Fields**: Use camelCase (e.g., `request.auth.claims`).
-
-* **Methods**: Use PascalCase (e.g., `request.header.Get("Content-Type")`).
-
-* **Utility Functions**: camelCase (e.g. trim) for the full list see [here](https://expr-lang.org/docs/language-definition).
-
-## Expression Context
-
-### **Request Object**
-
-The `request` object is a read-only entity that provides details about the incoming client request. It is accessible at all times during the lifecycle of the request.
-
-**Available Fields**:
-
-* `request.method`
-
-* `request.url.host`
-
-* `request.url.path`
-
-* `request.url.port`
-
-* `request.error`
-
-#### **Header Access**:
-
-* `request.header.Get('Content-Type')` – Header retrieval is case-insensitive.
-
-#### Example expressions
-
-```bash
+    condition: "request.header.Get('x-block-mutation') == 'yes'"
+```
+
+In this example, the condition is evaluated each time a request is made. Template Expressions provide access to request details, including headers, URLs, and query parameters. However, not all fields are consistently available throughout the request lifecycle. For instance, authentication data becomes accessible only after successful authentication.
+
+During the build process, the router validates expressions. If an expression is invalid or does not return the expected type (such as a boolean or string), an error message is generated to ensure clarity and correctness.
+
+### Template Language
+
+The expressions are based on the [expr-lang](https://github.com/expr-lang/expr) template language. This language is characterized by its:
```

Comment:
Did you auto format it?

Reply:
Mintlify seems to be auto converting stars to dashes
![image](https://github.com/user-attachments/assets/fcd515f1-5920-445c-b742-1e59cd41ab94)